griptape/drivers/web_search/exa_web_search_driver.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from attrs import define, field
+
+from griptape.artifacts import JsonArtifact, ListArtifact
+from griptape.drivers import BaseWebSearchDriver
+from griptape.utils import import_optional_dependency
+from griptape.utils.decorators import lazy_property
+
+if TYPE_CHECKING:
+    from exa_py.api import Exa
+
+
+@define
+class ExaWebSearchDriver(BaseWebSearchDriver):
+    api_key: str = field(kw_only=True)
+    _client: Exa = field(default=None, kw_only=True, alias="client", metadata={"serializable": False})
```

Comment:
You can remove `metadata={"serializable": False}` -- no metadata means no serialization.

Reply:
This is repeated in multiple other files: 

[Here](https://github.com/griptape-ai/griptape/blob/15907f049a9f9d23441a3b7f8ce4a5e29039afa8/griptape/drivers/sql/sql_driver.py#L19)
[Here](https://github.com/griptape-ai/griptape/blob/15907f049a9f9d23441a3b7f8ce4a5e29039afa8/griptape/tokenizers/google_tokenizer.py#L21)
And a couple of other places.